Ran into a different problem though at a later stage of the build:
```
# github.com/cockroachdb/cockroach/vendor/github.com/knz/go-libedit/unix
/usr/bin/ld: cannot find -ltinfo
collect2: error: ld returned 1 exit status

```

---







*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/19648#-:-KxiWJHCDuwzLa-S6Y71:br5xage)*
<!-- Sent from Reviewable.io -->

--- a/build/bootstrap/bootstrap-debian.sh
+++ b/build/bootstrap/bootstrap-debian.sh
@@ -5,9 +5,22 @@
 
 set -euxo pipefail
 
+curl -fsSL https://deb.nodesource.com/gpgkey/nodesource.gpg.key | sudo apt-key add -
+echo "deb https://deb.nodesource.com/node_6.x xenial main" | sudo tee /etc/apt/sources.list.d/nodesource.list
+
+curl -fsSL https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+
 sudo apt-get update
 sudo apt-get dist-upgrade -y
-sudo apt-get install -y --no-install-recommends docker.io git autoconf cmake
+sudo apt-get install -y --no-install-recommends \
+  autoconf \
+  cmake \
+  docker.io \
+  libtinfo-dev \
+  git \
+  nodejs \
+  yarn
 
 sudo adduser "${USER}" docker
 

